jni/rubicon.c

```diff
@@ -1022,8 +1025,9 @@ JNIEXPORT jobject JNICALL Java_org_beeware_rubicon_PythonInstance_invoke(JNIEnv
     jfieldID PythonInstance__id = (*env)->GetFieldID(env, PythonInstance, "instance", "J");
     LOG_D("id: %ld", (long)PythonInstance__id);
 
-    long instance = (*env)->GetLongField(env, thisObj, PythonInstance__id);
-    LOG_D("instance: %ld", instance);
+    jlong instance = (*env)->GetLongField(env, thisObj, PythonInstance__id);
+    // Since `jlong` is a signed 64-bit integer, we use a relevant macro.
+    LOG_D("instance: " PRId64, instance);
```

Comment:
Should this have a % in the string?

Reply:
You're right! I'll fix in a follow-up. (Earlier I somehow thought `PRId64` already had the `%` sign in it.)

In case you're curious:

```
➜  /tmp $ cat hmm.c
#define __STDC_FORMAT_MACROS
#include <inttypes.h>
#include <stdio.h>

int main(void) {
  uint64_t x = 31;
  printf("%" PRId64 "\n", x);
}
➜  /tmp $ gcc hmm.c -o hmm && ./hmm
31
```